src/App.js

```diff
@@ -4,21 +4,24 @@ import Profile from './Views/Profile';
 import Login from './Views/Login';
 import './App.css';
 import Layout from './Components/Layout';
+import WindowDimensionsProvider from './utils/WindowDimensionsProvider';
```

Comment:
I know I said we needed to be more explicit but isn't this name super specific? haha Almost like an ultimate attack. ***insert yugi & kaiba attack meme**

Reply:
The name needs to be like that because it describes exactly what the component does; it is a context provider that gives the window's dimensions to the whole app, there may be more context providers later on so we can come up with a different naming by then, maybe by changing the file structure and having a folder called providers with the appropriately named components inside.

Reply:
Seems fair. I think that's important, have the awareness that in future may change, but for now it completely works the way it is. When the time comes, hope we refrain from adding another word haha the file structure change sounds good.